miner/worker.go

```diff
@@ -70,6 +70,12 @@ const (
 
 	// the default to wait for the mev miner to finish
 	waitMEVMinerEndTimeLimit = 50 * time.Millisecond
+
+	// Reserve block size for the following 3 components:
+	// a. System transactions at the end of the block
+	// b. Seal in the block header
+	// c. Overhead from RLP encoding
+	blockReserveSize = 100 * 1024
```

Comment:
Do we have a measurement/estimate to know upper limit of these 3 components to know if `100 * 1024` should be enough?

Reply:
a. system transactions , no one have big calldata,  size is less than 1k, for example, https://bscscan.com/block/42125600
b. Seal is 32 byte
c. rpl encode has a very low overhead

blockReserveSize `3 *1024`, I think is enough,
so set it  to `100 * 1024` ensure security is ok, IMO.